### ISSUE TITLE
Provide session expired message

### ISSUE THREAD

**Issue by ItsKarma** [DeployBoard/deployboard-python#43]
Right now if your session expires, we redirect back to the login page while clearing the flask session. We should pass an info message on that page saying the session expired, please log in again. Right now it's unclear as to why the user is being returned to the login page.

**Comment by OBITORASU** [comment]
Adding a ```logger.info``` to the body of the ```if session['exp'] < datetime.now().timestamp():``` statement in the ```
deployboard/src/web/main.py ``` should fix this issue right? If I am going in the right direction, I would love to work on this, if anything is wrong with this idea do let me know as well.

**Comment by ItsKarma** [comment]
Thanks for taking a look. I think that would be fine to do, but doesn't solve the original issue, which I probably could have been a bit more clear about. Adding a `logger.info` would provide some information to an administrator, but the end user wouldn't have access to the system logs, so they would never see the message.

I think what we would need to do is in that same block you referenced, but where we redirect to the login page. Also pass a parameter into the login page indicating a session has expired, something like: `return redirect(url_for('login_page.login', expired=true))`. This would make the request url `/login/?expired=true`. _notice I'm changing logout to login, since the redirect to the logout page is unnecessary since we are already clearing the session._ Then we would need to make another update on the `webroutes/login.py` to check for this query string parameter and pass it into the login page template.

We would then need to make a change to the template to pop a message (similar to how we pass in an error message) that says something like "Session expired, please log in again."

TLDR: 3 tings:
1. Update main.py to redirect to login page with an expired message (current thought is query string paramter).
2. Update login.py to check for that message, and pass into template if found.
3. Update login.html template to support the expired message.

**Comment by OBITORASU** [comment]
I think my current technical stack is not good enough for this issue. I guess I will pass on this issue over to someone better suited to handle it. Thanks for your taking your time off to explain the issue means a lot.
